mteb/tasks/BitextMining/multilingual/NTREXBitextMining.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import annotations
+
+from typing import Any
+
+from datasets import DatasetDict, load_dataset
+
+from mteb.abstasks import AbsTaskBitextMining, CrosslingualTask
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+# Mapping from ISO 639-3 + script to ISO 639-1 used in NTREX
+_ISO6393_to_ISO6391 = {
+    "ara_Arab": "ar",
+    "deu_Latn": "de",
+    "eng_Latn": "en",
+    "spa_Latn": "es",
+    "fra_Latn": "fr",
+    "hin_Deva": "hi",
+    "ind_Latn": "id",
+    "ita_Latn": "it",
+    "jpn_Jpan": "ja",
+    "kor_Hang": "ko",
+    "por_Latn": "pt",
+    "rus_Cyrl": "ru",
+    "tha_Thai": "th",
+    "tur_Latn": "tr",
+    "vie_Latn": "vi",
+    "zho_Hans": "zh",
+}
+
+_SPLIT = ["train"]
+
+# number of sentences to use for evaluation
+_N = 1997
+
+
+def extend_lang_pairs() -> dict[str, list[str]]:
+    # add all possible language pairs
+    hf_lang_subset2isolang = {}
+    for x in _ISO6393_to_ISO6391.keys():
+        if "-" not in x:
+            for y in _ISO6393_to_ISO6391.keys():
+                if x != y:
+                    pair = f"{x}-{y}"
+                    hf_lang_subset2isolang[pair] = [
+                        x.replace("_", "-"),
+                        y.replace("_", "-"),
+                    ]
+    return hf_lang_subset2isolang
```

Comment:
Following my comment in PR #386, if it's the same funciton we should think about not duplicating the code.

Reply:
(see comment my comment on the PR)